extensions/schema-compare/src/schemaCompareResult.ts

```diff
@@ -16,7 +16,9 @@ const diffEditorTitle = localize('schemaCompare.ObjectDefinitionsTitle', 'Object
 const applyConfirmation = localize('schemaCompare.ApplyConfirmation', 'Are you sure you want to update the target?');
 const reCompareToRefeshMessage = localize('schemaCompare.RecompareToRefresh', 'Press Compare to refresh the comparison.');
 const generateScriptEnabledMessage = localize('schemaCompare.generateScriptEnabledButton', 'Generate script to deploy changes to target');
+const generateScriptNoChangesMessage = localize('schemaCompare.generateScriptNoChanges', 'No changes to generate script for');
```

Comment:
Is this complete? It seems to trail off

Reply:
Changed to "No changes to generate script for." Is there a better way to phrase this?